### PR TITLE
Feature/notification item

### PR DESCRIPTION
**Pull request by Kamchybekk**

### CODE REVIEW

File: src/Components/UI/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled from "styled-components";
+
+const Btn = ({ title, onClick, color }) => {
```
Comment (songhee24) [inline src/Components/UI/Button.jsx]: Maybe you should call this Button ?

File: src/Components/UI/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <Button onClick={onClick} color={color}>
```
Comment (songhee24) [inline src/Components/UI/Button.jsx]: And Why are you created the button is the branch called "notification-item"

File: src/Components/UI/NotificationItem.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+const NotificationItem = ({ avatar, date, title }) => {
+  return (
+    <StyledNotificationItem>
+      <div>
+        <BlueIcon />
+        <UserAvatar />
+        <UserName />
+      </div>
+      <span>Moved to list Done</span>
+      <DateAdded>Sep 15 at 13:23 PM</DateAdded>
```
Comment (songhee24) [inline src/Components/UI/NotificationItem.jsx]: should we use another package for username and useravatar components ?

File: src/Components/UI/NotificationItem.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+import UserAvatar from "./UserAvatar";
+import UserName from "./UserName";
+import arrowSvg from "../../assets/icons/arrow.svg";
+
+const NotificationItem = ({ avatar, date, title }) => {
```
Comment (songhee24) [inline src/Components/UI/NotificationItem.jsx]: NotificationItem stores only user's info ? or something else ?

**Review by songhee24 — CHANGES_REQUESTED** [review]

File: src/Components/UI/Button.jsx (hunk 1 above)
Comment (Kamchybekk) [reply src/Components/UI/Button.jsx]: > 

Fixed it in the feature/Button branch. But in general, there should not have been a button

File: src/Components/UI/Button.jsx (hunk 2 above)
Comment (Kamchybekk) [reply src/Components/UI/Button.jsx]: > 

removed from notification item branch

File: src/Components/UI/NotificationItem.jsx (hunk 3 above)
Comment (Kamchybekk) [reply src/Components/UI/NotificationItem.jsx]: Кийин пользовательдин аватары менен ник неймин в будущем башка жерден дагы колдонобузда ошол учун озунчо жазгам

File: src/Components/UI/NotificationItem.jsx (hunk 4 above)
Comment (Kamchybekk) [reply src/Components/UI/NotificationItem.jsx]: ?

File: src/Components/UI/NotificationItem.jsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
+import React from "react"
+import styled from "styled-components"
+import UserAvatar from "./UserAvatar"
+import UserName from "./UserName"
+import arrowSvg from "../../assets/icons/arrow.svg"
+import png from "../../assets/images/Rectangle.png"
+
+const NotificationItem = ({ titleBoard, nameColumn }) => {
```
Comment (songhee24) [inline src/Components/UI/NotificationItem.jsx]: Screen Shot 2022-10-10 at 11 53 03
THATS ALSO PROPS IN NOTIFICATION ITEM

**Review by songhee24 — CHANGES_REQUESTED** [review]

**Pull request merged**